src/components/TouchableWithoutFeedback.jsx

```diff
@@ -20,8 +20,8 @@ const TouchableWithoutFeedback = ({ onPress, onPressIn, ...props }) => {
   function _onPress(e) {
     const { pageX, pageY } = e.nativeEvent;
 
-    const absX = Math.abs(_touchActivatePositionRef.current.pageX - pageX);
-    const absY = Math.abs(_touchActivatePositionRef.current.pageY - pageY);
+    const absX = Math.abs(_touchActivatePositionRef?.current?.pageX - pageX);
```

Comment:
@muhammedjasircp what about https://github.com/bigbinary/neeto-ui-rn/blob/25decf8d18b612463e648d6bff279f72e34daa5c/src/components/TouchableWithoutFeedback.jsx#L12

Reply:
got it, invalid comment. merging.

Reply:
pageX and pageY won't be undefined in nativeEvent. So when we are setting them that won't be an issue, right?